models/engine/file_storage.py

```diff
@@ -35,6 +36,15 @@ def all(self, cls=None):
             return {k: v for k, v in self.__objects.items()
                     if v.__class__ == cls}
 
+    def count(self, cls=None):
+        """Return number of objects in storage"""
+        return len(self.all(cls))
+
+    def get(self, cls, id):
+        """Return object based on it class name and id"""
+        key = cls + "." + id
+        return self.all().get(key, None)
```

Comment:
Same here as for db_storage `get`. You don't need to copy the whole dictionary just to get a single entry. You can also just call `get` on `self.__objects`.